src/Types/Workspace/Builder.php

```diff
@@ -162,7 +162,17 @@ public function build(Environment $environment, DefinitionCollection $definition
                         echo $attribute . "\n";
                     }
                 } else {
-                    echo "attribute value:\n\n";
+                    echo "attribute type:\n\n  ";
```

Comment:
Would it be sensible to have the `type` value printed inline with `attribute type:` ?

..so that the output can be grepped by `attribute type` string and then the actual type value can be extracted

Reply:
Done